MyLibrary/Sources/ScheduleFeature/Detail.swift

```diff
@@ -63,6 +63,8 @@ public struct ScheduleDetail {
 public struct ScheduleDetailView: View {
 
   @Bindable public var store: StoreOf<ScheduleDetail>
+  
+  @Environment(\.openURL) var openURL
```

Comment:
Please use @Dependency(\.openURL) instead for Test dependency in Reducer.

Reply:
If I use @Dependency, I don't need to switch os in view because reducer can run openURL. Is it right?
![CleanShot 2024-03-14 at 14 07 49@2x](https://github.com/tryswift/trySwiftTokyoApp/assets/2066707/b00a0f7d-fdee-4907-89d5-ec9338ff62a2)

Reply:
I tried to apply this pattern.
https://github.com/tryswift/trySwiftTokyoApp/pull/24/commits/2133159fa32e25689525ac9f9d4e2570527506ef